ibm_catalog.json

```diff
@@ -39,8 +39,8 @@
           "title": "Configures Cloud Monitoring. instance and agents."
         },
         {
-          "description": "Configures Activity Tracker routing",
-          "title": "Configures an Activity Tracker route with a COS bucket target."
+          "description": "Configures Activity Tracker event routing",
+          "title": "Configures an Activity Tracker event routing to a COS bucket and optionally to a Log Analysis instance."
```

Comment:
Lets not add Log Analysis here because here the default features are listed.

Reply:
resolved